scripts/kind-e2e/e2e.sh

```diff
@@ -176,29 +176,45 @@ function get_globalip() {
     echo $gip
 }
 
-function test_connection() {
-    if [[ $globalnet = true ]]; then
-        nginx_svc_ip_cluster3=$(get_globalip nginx-demo cluster3)
-    else
-        nginx_svc_ip_cluster3=$(kubectl --context=cluster3 get svc -l app=nginx-demo | awk 'FNR == 2 {print $3}')
-    fi
-    if [[ -z "$nginx_svc_ip_cluster3" ]]; then
-        echo "Failed to get nginx-demo IP"
-        exit 1
-    fi
+function try_connect() {
+    target=$1
     netshoot_pod=$(kubectl --context=cluster2 get pods -l app=netshoot | awk 'FNR == 2 {print $1}')
 
-    echo "Testing connectivity between clusters - $netshoot_pod cluster2 --> $nginx_svc_ip_cluster3 nginx service cluster3"
+    echo "Testing connectivity between clusters - $netshoot_pod cluster2 --> $target nginx service cluster3"
 
     attempt_counter=0
     max_attempts=5
-    until $(kubectl --context=cluster2 exec ${netshoot_pod} -- curl --output /dev/null -m 30 --silent --head --fail ${nginx_svc_ip_cluster3}); do
+    until $(kubectl --context=cluster2 exec ${netshoot_pod} -- curl --output /dev/null -m 30 --silent --head --fail ${target}); do
         if [[ ${attempt_counter} -eq ${max_attempts} ]];then
           echo "Max attempts reached, connection test failed!"
           exit 1
         fi
         attempt_counter=$(($attempt_counter+1))
     done
+}
+
+function test_connection() {
+    if [[ $globalnet = true ]]; then
+        nginx_svc_ip_cluster3=$(get_globalip nginx-demo cluster3)
+    else
+        nginx_svc_ip_cluster3=$(kubectl --context=cluster3 get svc -l app=nginx-demo | awk 'FNR == 2 {print $3}')
+    fi
+    if [[ -z "$nginx_svc_ip_cluster3" ]]; then
+        echo "Failed to get nginx-demo IP"
+        exit 1
+    fi
+    try_connect $nginx_svc_ip_cluster3
+    if [[ $lighthouse = true ]]; then
+        try_connect nginx-demo
+        resolved_ip=$(kubectl --context=cluster2 exec ${netshoot_pod} -- ping -c 1 -W 1 nginx-demo | grep PING | awk '{print $3}')
+        # strip the () braces from resolved_ip
+        resolved_ip=${resolved_ip:1:-1}
```

Comment:
Can you check why we are seeing this line in travis.
"command terminated with exit code 1"

https://travis-ci.com/github/submariner-io/submariner-operator/jobs/298879463#L6358

Reply:
This is because we're using ping and ping to the pod fails. I couldn't figure out how to get IP from curl, so ended up using ping. Let me see if use something else. `dig` might work.

Reply:
It is coz ping to service will fail. Updated to `/dev/null` the error.